packages/path-inputs/src/button.tsx

```diff
@@ -39,14 +40,24 @@ export interface ButtonProps {
      * @default horizontal
      */
     layout?: 'horizontal' | 'vertical';
```

Comment:
when you provide `defaultProps` static property, the interface should have a regular property (not optional). this is because when you set `layout?` you indicate that if not provided it will be `undefined`.

You should remove all `?` suffix. once you provide `defaultProps` typescript and the react integration takes care of it on you behalf. Try it out and then remove those values. 

btw, you should keep the actual default value in the comments because I think storybook docs doesn't resolve it as `react` does